wagtail/admin/templates/wagtailadmin/generic/index_results.html

```diff
@@ -2,6 +2,10 @@
 {% load i18n wagtailadmin_tags %}
 
 {% block before_results %}
+    {% if view.active_filters %}
+        {% include "wagtailadmin/shared/active_filters.html" with active_filters=view.active_filters %}
+    {% endif %}
+
```

Comment:
I figured we _didn't_ want to include the active filters block on listings that use the old-style page furniture (such as reports) where the filter form is always visible, and rather than go through the 'is breadcrumb_items' dance to decide whether that's the case or not, I thought it would be cleaner to add it individually as and when those views get ported over.